docs/ReferenceManyToManyInput.md

```diff
@@ -7,6 +7,12 @@ title: "The ReferenceManyToManyInput Component"
 
 This [Enterprise Edition](https://marmelab.com/ra-enterprise)<img class="icon" src="./img/premium.svg" /> component allows to create, edit or remove relationships between two resources sharing an associative table. The changes in the associative table are sent to the dataProvider when the user submits the form, so that they can cancel the changes before submission.
 
+<video controls autoplay playsinline muted loop width="100%">
+  <source src="./img/reference-many-to-many-input.webm" type="video/webm" />
+  <source src="./img/reference-many-to-many-input.mp4" type="video/mp4" />
+  Your browser does not support the video tag.
+</video>
+
 In this example, `artists.id` matches `performances.artist_id`, and `performances.event_id` matches `events.id`:
```

Comment:
It would be great if the model matched the one for ReferenceManyToManyField (with band instead of artist)